src/reflect/scala/reflect/internal/Trees.scala

```diff
@@ -205,11 +205,8 @@ trait Trees extends api.Trees {
       new ForeachPartialTreeTraverser(pf).traverse(this)
     }
 
-    def changeOwner(pairs: (Symbol, Symbol)*): Tree = {
```

Comment:
Let's leave this version around as a deprecated overload, it is likely to be called from compiler plugins.